app/src/main/java/com/ej/rovadiahyosefcalendar/classes/SiddurMaker.java

```diff
@@ -532,9 +532,57 @@ public ArrayList<HighlightString> getShacharitPrayers() {
         addAmidahToSiddur(false, false);
 
         if (jewishDateInfo.getJewishCalendar().isRoshChodesh()
-                || jewishDateInfo.getJewishCalendar().isCholHamoed()
-                || jewishDateInfo.getJewishCalendar().isChanukah()) {//TODO fix when it is chanukah and rosh chodesh
+         || jewishDateInfo.getJewishCalendar().isCholHamoed()
+         || jewishDateInfo.getJewishCalendar().isChanukah()) {//TODO fix when it is chanukah and rosh chodesh
             addHallel();
+            if (jewishDateInfo.getJewishCalendar().getJewishMonth() == JewishCalendar.TISHREI && jewishDateInfo.getJewishCalendar().isCholHamoed()) {
```

Comment:
There is a isCholHamoedSuccos method, so this can be simplified. I will update it.